docusaurus/docs/Android/03-ui/06-guides/09-customizing-message-list-scroll-behavior.mdx

```diff
@@ -1,3 +1,6 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Customizing MessageListView Scrolling Behavior
 
 ## Introduction
```

Comment:
There's a snippet from `MessageListScrollHelper` below. I think it doesn't require the Kotlin version as we are referring to an actual code in our code base.